backend/db/schema.sql

```diff
@@ -0,0 +1,30 @@
+--- dropoff data
+CREATE TABLE dropoffs (
+    id integer PRIMARY KEY SERIAL,
+    state_name varchar(20) NOT NULL,
```

Comment:
This should be 
```suggestion
    state_name varchar(12) NOT NULL,
```

as the longest state name is **Rhode Island** with 12 characters

Reply:
North Carolina has 14, so does its neighbor to the south, South Carolina.
West Virginia has 13

don't sleep on the southern atlantic coast they are nice :-D

Reply:
If California splits North California will have a whopping 16 chars!

i hope ballot nav lives a long time and we have know way of knowing what could happen, so why not spare some extra bytes :laughing:

Reply:
Washington D.C. has 15!

Reply:
Hmm -- Google Search has failed me